android/app/src/main/java/org/openbot/common/FeatureList.java

```diff
@@ -71,7 +71,8 @@
     subCategories = new ArrayList<>();
     subCategories.add(new SubCategory(FREE_ROAM, R.drawable.ic_game, "#FFFF6D00"));
     subCategories.add(new SubCategory(DATA_COLLECTION, R.drawable.ic_storage, "#93C47D"));
-    subCategories.add(new SubCategory(AI_MODELS, R.drawable.ic_person_search, "#FFD966"));
+    subCategories.add(new SubCategory(AI_MODELS, R.drawable.ic_person_search, "#4B7BFF"));
```

Comment:
refactor to autopilot and find a suitable icon

Reply:
done. not sure about the icon though. inputs welcome